main/webapp/modules/core/scripts/index/parser-interfaces/preview-table.js

```diff
@@ -37,6 +37,8 @@ Refine.PreviewTable = function(projectData, elmt) {
   this._render();
 };
 
+var controlCharacters = ["NUL", "SOH", "STX", "ETX", "EOT", "ENQ", "ACK", "BEL", "BS", "TAB", "LF", "VT", "FF", "CR", "SO", "SI", "DLE", "DC1", "DC2", "DC3", "DC4", "NAK", "SYN", "ETB", "CAN", "EM", "SUB", "ESC", "FS", "GS", "RS", "US", "SPACE"];
```

Comment:
Let's replace `SPACE` with just `SP` to reduce verbosity

Reply:
yeah, was thinking that as well. I will replace that now.